src/Robo/Plugin/Enums/LocalDevEnvironmentTypes.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Usher\Robo\Plugin\Enums;
+
+enum LocalDevEnvironmentTypes: string
```

Comment:
Big picture idea. Should each of these environment types be classes that extend a base class? Then they could override methods as needed too, and encapsulate all of their custom needs?

Reply:
I see where you are going and I like it if I thought we would keep supporting both long-term but I can also imagine us dropping Lando support if we move everything over. I don't want to over-architect things at this stage until we figure out the direction we are moving.

Reply:
👍🏼 Makes sense.